get-wget-lua.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+#
+# This script downloads and compiles wget-lua.
+#
+
+# first, try to detect gnutls or openssl
+CONFIGURE_SSL_OPT=""
+if builtin type -p pkg-config &>/dev/null
+then
+  if pkg-config gnutls
+  then
+    echo "Compiling wget with GnuTLS."
+    CONFIGURE_SSL_OPT="--with-ssl=gnutls"
+  elif pkg-config openssl
+  then
+    echo "Compiling wget with OpenSSL."
+    CONFIGURE_SSL_OPT="--with-ssl=openssl"
+  fi
+fi
+
+WGET_DOWNLOAD_URL="http://warriorhq.archiveteam.org/downloads/wget-lua/wget-1.14.lua.LATEST.tar.bz2"
```

Comment:
Sure, I'll fix that now. I blindly copied this file from some archive project repo